styles/partials/base-partials/_utils.scss

```diff
@@ -0,0 +1,98 @@
+//not sure this is needed
+// @import "../mixins";
+
+/* alignment
+......................................*/
+.text-top {
+  vertical-align: top !important;
+}
+.text-middle {
+  vertical-align: middle !important;;
+}
+.text-bottom {
+  vertical-align: bottom !important;;
+}
+
+/* display
+......................................*/
+.block {
+  display: block !important;
+}
+.inline {
+  display: inline-block !important;
```

Comment:
Perhaps we should audit all the files that we should include for now, all of this is from the existing Quartz CSS.  :)